src/components/TimeGraph.vue

```diff
@@ -316,17 +325,21 @@ export default {
       showEventPopUp: false,
       showEventDialogue: false,
       showHelpDialogue: false,
-      personYears: store.getters.getTimeline,
+      // personYears: store.getters.getTimeline,
       showEventDisplay: false,
       burgerMenuActive: false,
       showCreateBiograph: !store.getters.getPersonCreated,
       newDimDetails: {
         title: "",
       },
+      lang: store.state.settings.language,
```

Comment:
lang is in data == a ref() variable that is a local copy of the language setting --> replace with a computed 

currently not a problem but it would fail to update the dropdown, if the setting was changed elsewhere

Reply:
see <https://vuejs.org/guide/essentials/computed.html#writable-computed>